satpy/tests/test_cf_roundtrip.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
Do we maybe want to start not including these in new modules given the conversation in pyresample with @gerritholl?